src/User/Infra/Security/Oauth/EavSecurityUserProvider.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\User\Infra\Security\Oauth;
+
+use HWI\Bundle\OAuthBundle\OAuth\ResourceOwnerInterface;
+use HWI\Bundle\OAuthBundle\OAuth\Response\UserResponseInterface;
+use HWI\Bundle\OAuthBundle\Security\Core\User\OAuthAwareUserProviderInterface;
+use MsgPhp\Domain\Exception\EntityNotFoundException;
+use MsgPhp\Domain\Factory\EntityAwareFactoryInterface;
+use MsgPhp\Domain\Message\{DomainMessageBusInterface, MessageDispatchingTrait};
+use MsgPhp\Eav\Entity\Attribute;
+use MsgPhp\User\Command\{AddUserAttributeValueCommand, ConfirmUserCommand, CreateUserCommand};
+use MsgPhp\User\Entity\{User, UserAttributeValue};
+use MsgPhp\User\Infra\Security\SecurityUserProvider as BaseSecurityUserProvider;
```

Comment:
there's no need for an alias here

Reply:
`SecurityUserProvider` already exists in the current namespace, but we want to inject `SecurityUserProvider` from the parent namespace.

but im leaning to move the entire Oauth namespace to Infra/HWIOAuth

Reply:
if are not using the `SecurityUserProvider` in the current namespace you don't have to use an alias.